content/en/docs/Kyverno CLI/_index.md

```diff
@@ -223,3 +223,156 @@ policies:
 ```
 kyverno apply /path/to/add_network_policy.yaml --resource /path/to/required_default_network_policy.yaml -f /path/to/value.yaml
 ```
+
+#### Policy Report:
+
+Use '--policy_report' with apply command to generate policy report.
+
+Input combination:
+
+| S.No | policy        | resource         | cluster   | namespace      | interpretation                                                                           |
+| ---- |:-------------:| :---------------:| :--------:| :-------------:| :----------------------------------------------------------------------------------------| 
+| 1.   | policy.yaml   | -r resource.yaml | false     |                | apply policy from file to the resource from file                                         |
+| 2.   | policy.yaml   | -r resourceName  | true      |                | apply policy from file to the resource in cluster                                        |
+| 3.   | policy.yaml   |                  | true      |                | apply policy from file to all the resources in cluster                                   |
+| 4.   | policy.yaml   | -r resourceName  | true      | -n=namespace   | apply policy from file to the resource in cluster in mentioned namespace                 |
+| 5.   | policy.yaml   |                  | true      | -n=namespace   | apply policy from file to all the resources in cluster in mentioned namespace            |
+
+
+Example:
+
+Consider the following policy and resources:
+
+policy.yaml
+```apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: require-pod-requests-limits
+  annotations:
+    policies.kyverno.io/category: Workload Management
+    policies.kyverno.io/description: As application workloads share cluster resources, it is important 
+      to limit resources requested and consumed by each pod. It is recommended to require 
+      'resources.requests' and 'resources.limits' per pod. If a namespace level request or limit is 
+      specified, defaults will automatically be applied to each pod based on the 'LimitRange' configuration.
+spec:
+  validationFailureAction: audit
+  rules:
+  - name: validate-resources
+    match:
+      resources:
+        kinds:
+        - Pod
+    validate:
+      message: "CPU and memory resource requests and limits are required"
```

Comment:
Also valid:
```suggestion
      message: CPU and memory resource requests and limits are required"
```

Reply:
@sftim didn't get what were you suggesting here. 
this should be fine right. 
```message: "CPU and memory resource requests and limits are required"```. Since we have more than a word, its better to keep the enclosing quotes.

Reply:
Either's fine. YAML lets you have strings without quoting them. No need to modify anything here.